store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreTest.java

```diff
@@ -255,12 +255,22 @@ public void shouldNotAllowOverwritingOfGraphWithinFederatedScope() throws Except
             assertContains(e, "GraphId: ", ACC_ID_2);
         }
     }
-
-    @Test(expected = UnsupportedOperationException.class)
-    public void shouldDoUnhandledOperation() throws Exception {
-        store.doUnhandledOperation(null, null);
+    @Test
+    public void shouldThrowAppropriateExceptionWhenHandlingAnUnsupportedOperation() {
+        // Given
+        Operation op = new OperationImpl();
```

Comment:
Ah - just spotted that you'll need to add some imports to get this to work

Reply:
Add 
`import uk.gov.gchq.gaffer.operation.impl.OperationImpl;` between the OperationException and AddElements imports to get this to compile

Reply:
Should be fine now